skbeam/core/tests/test_correlation.py

```diff
@@ -102,7 +115,20 @@ def test_lazy_multi_tau():
     for second_half_result in gen_second_half:
         pass
 
-    assert np.all(full_result.g2 == second_half_result.g2)
+    assert np.all(full_result.correlation_results ==
+                  second_half_result.correlation_results)
+
+
+def test_two_time_corr():
+    y = []
```

Comment:
call `setup()` as the first line here. I am not sure how nosetests determines the order in which tests are run.  If this test is ever run first, there will be cryptic errors.